packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -423,25 +492,58 @@ export class SeedlessOnboardingController extends BaseController<
    * - The password is incorrect (from encryptor.decrypt)
    * - The decrypted vault data is malformed
    */
-  async #unlockVaultWithPassword(password: string): Promise<{
+  async #unlockVaultAndGetBackupEncKey(): Promise<{
```

Comment:
it would be rather better to have password as optional param here, rather thn loading it from state inside the function